Sdk/Enums/CurrencyCodeEnum.cs

```diff
@@ -33,6 +33,179 @@ public enum CurrencyCodeEnum
         /// <summary>
         /// The usd
         /// </summary>
-        Usd = 840
```

Comment:
Hi, can you just change the name of `Usd` to `USD` to have unified naming? Even it is little breaking change, it should be ok.